src/main/java/Milo/TaskList.java

```diff
@@ -0,0 +1,28 @@
+package Milo;
+
+import Milo.TaskObj.Task;
+import java.util.ArrayList;
+
+/*
+* Represents task list object
+ */
+public class TaskList {
+
+    private final ArrayList<Task> todoList;
```

Comment:
Would it be better to just name your ArrayList<Task> attribute here as taskList because there is a task type named Todos but taskList stores other tasks like Deadline tasks and Event tasks as well?